src/services/CertificationService.ts

```
@@ -0,0 +1,106 @@
+import { CertificateDetail } from "@/components/ui/certificate";
+
+export class CertificationService {
+  private static readonly BASE_URL = "http://localhost:8080/api";
+
+  static async getCertificates(userId: string, token: string) {
+    try {
+      const res = await fetch(`${this.BASE_URL}/api/certificates/${userId}`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)